deltachat-ios/Controller/DocumentGalleryController.swift

```diff
@@ -41,9 +42,16 @@ class DocumentGalleryController: UIViewController {
                 self?.redirectToMessage(of: indexPath)
             }
         )
+        let shareItem = ContextMenuProvider.ContextMenuItem(
+            title: String.localized("menu_share"),
+            imageName: "square.and.arrow.up",
+            action: #selector(DocumentGalleryFileCell.share(_:)), onPerform: { [weak self] indexPath in
+                self?.shareAttachment(of: indexPath)
+            }
+        )
 
         let menu = ContextMenuProvider()
-        menu.setMenu([showInChatItem, deleteItem])
+        menu.setMenu([showInChatItem, deleteItem, shareItem])
```

Comment:
```suggestion
        menu.setMenu([showInChatItem, shareItem, deleteItem])
```

delete, as destructing item, should be last in menu for internal and external consistency.